### ISSUE TITLE
REPL: "noop" is not exported by "https://unpkg.com/svelte@3.17.1/internal/index.js"

### ISSUE THREAD

**Issue by powellnorma** [sveltejs/sites#553]
Full error:
```
"noop" is not exported by "https://unpkg.com/svelte@3.17.1/internal/index.js", imported by "https://unpkg.com/svelte@3.17.1/store/index.mjs".
```

Happens e.g. here:
https://svelte.dev/repl/6dfa9e69a65243b0bbc44d66d91b8325?version=3.17.1

But not on newer versions:
https://svelte.dev/repl/6dfa9e69a65243b0bbc44d66d91b8325?version=4.2.11

![image](https://github.com/sveltejs/sites/assets/101364699/2691b9ec-e860-40da-bdd8-ad8d33173e62)

**Comment by Conduitry** [comment]
This was fixed in Svelte 3.29.5 in https://github.com/sveltejs/svelte/pull/5624 over three years ago. I don't think we need to make the REPL's bundling code still accommodate the broken packages in older versions of Svelte 3.

**Comment by powellnorma** [comment]
@Conduitry  I see, however please note that this often means that older links just seem to be "broken", and there is no hint of "try again with newer version". E.g. google often still references links:

![image](https://github.com/sveltejs/sites/assets/101364699/3b586cbd-d50c-453c-97a7-3fae3f89f189)

Or GitHub Discussions, e.g. basically _all_ links used here: https://github.com/sveltejs/svelte/issues/4285
 
Perhaps as a simple workaround we could just automatically use the next nearest working svelte version in such cases?


It could (potentially) be interesting to support older versions when wanting to compare behavior changes of different svelte versions, though.
